src/user_guide/001_Personium_Architecture.md

```diff
@@ -21,7 +21,8 @@ Personium defines the following three layered basic objects in order to achieve
 
 ### Cell  
 
-* A cell is a fundamental concepts of Personium
+* Cell is a data store for each data subject. For personal use, it will be PDS (Personal Data Store).
+* In Personium, the concept of data subject is extended to not only people but also organizations and things, so it can be used as a data store for organizations and things.
```

Comment:
```suggestion
* In Personium, the concept of data subject is extended to not only people but also organizations and objects, so it can be used as a data store for a NGO or a car.
```